### PR TITLE
Add duplicity 0.6.25 for legacy version.

### PR DESCRIPTION
**Pull request by lwis**

### CODE REVIEW

File: duplicity06.rb [hunk 1]
```diff
@@ -0,0 +1,227 @@
+class Duplicity < Formula
+  homepage "http://www.nongnu.org/duplicity/"
+  url "http://code.launchpad.net/duplicity/0.6-series/0.6.25/+download/duplicity-0.6.25.tar.gz"
+  sha1 "fe0b6b0b0dc7dbc02598d96567954b48c4308420"
+  revision 1
+
+  bottle do
```
Comment (DomT4) [inline duplicity06.rb]: Kill this block.

File: duplicity06.rb [hunk 2]
```diff
@@ -0,0 +1,227 @@
+class Duplicity < Formula
+  homepage "http://www.nongnu.org/duplicity/"
+  url "http://code.launchpad.net/duplicity/0.6-series/0.6.25/+download/duplicity-0.6.25.tar.gz"
+  sha1 "fe0b6b0b0dc7dbc02598d96567954b48c4308420"
+  revision 1
```
Comment (DomT4) [inline duplicity06.rb]: Kill this line.

File: duplicity06.rb [hunk 3]
```diff
@@ -0,0 +1,227 @@
+class Duplicity < Formula
+  homepage "http://www.nongnu.org/duplicity/"
+  url "http://code.launchpad.net/duplicity/0.6-series/0.6.25/+download/duplicity-0.6.25.tar.gz"
+  sha1 "fe0b6b0b0dc7dbc02598d96567954b48c4308420"
```
Comment (DomT4) [inline duplicity06.rb]: Bump this to SHA256.

File: duplicity06.rb [hunk 4]
```diff
@@ -0,0 +1,227 @@
+class Duplicity < Formula
```
Comment (DomT4) [inline duplicity06.rb]: You'll need to adjust the ` class ` name to match the new formula name. ` Duplicity06 `.

File: duplicity06.rb [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+  option :universal
+
+  # generated with homebrew-pypi-poet from
+  # for i in boto pyrax dropbox mega.py paramiko pycrypto
+  # lockfile python-swiftclient python-keystoneclient; do poet -r $i >>
+  # resources; done
+
+  resource "lockfile" do
+    url "https://pypi.python.org/packages/source/l/lockfile/lockfile-0.10.2.tar.gz"
+    sha1 "1df8b1fad0c344230eaa7ce5fbf06521a74d7a6b"
```
Comment (DomT4) [inline duplicity06.rb]: Likewise, all these need to be SHA256 here.

File: duplicity06.rb [hunk 6]
```diff
@@ -0,0 +1,227 @@
+class Duplicity < Formula
+  homepage "http://www.nongnu.org/duplicity/"
+  url "http://code.launchpad.net/duplicity/0.6-series/0.6.25/+download/duplicity-0.6.25.tar.gz"
```
Comment (DomT4) [inline duplicity06.rb]: This should be ` https:// `

**Comment by lwis** [comment]
@DomT4 thanks for the advice, changes made as requested.

**Comment by DomT4** [comment]
Merged! Thanks for making the changes, and your contribution to Homebrew. We appreciate it! :smiley_cat:. For future reference the preferred commit message for new formulae is ` ducks12 1.2.3 (new formula) `, I fixed that prior to merge here.

**Pull request closed without merging**